C/csb/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "csb"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/fcdimitr/CSB.git", "c53ee56e34f22f8e793ca45349fdcaa36e35f8af")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd csb/
+apk add meson --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
```

Comment:
We already have meson, is that not enough?

Reply:
I followed the same setup as 

https://github.com/JuliaPackaging/Yggdrasil/blob/d00e02ae9d5cc7ae96ffb798fc3a38cd8b3d9e2b/S/sgtsnepi/build_tarballs.jl

since it has same dependencies and build requirements.

Reply:
We have updated the rootfs in the last 13 months

Reply:
Oh, I see. I will fix it. Thanks!